public/sass/components/components.chat.scss

```diff
@@ -32,6 +32,15 @@
     padding-right: .5rem;
     overflow-y: scroll;
     overflow-x: hidden;
+
+    &--with-sticky::-webkit-scrollbar-track {
+      margin-top: 2rem;
+    }
+  }
+
+  &__start {
+    margin: 100px auto;
```

Comment:
I would have used flex here to do the centering but let's say this is OK as well :see_no_evil:

Reply:
Let's not put the horse in front of the karutza. We can improve if needed 😅